packages/cli/src/commands/verify.ts

```diff
@@ -0,0 +1,84 @@
+import type { CommandModule } from "yargs";
+import { verify } from "../verify";
+import { logError } from "../utils/errors";
+import { loadConfig } from "@latticexyz/config/node";
+import { WorldConfig, resolveWorldConfig } from "@latticexyz/world/internal";
+import { worldToV1 } from "@latticexyz/world/config/v2";
+import { getOutDirectory, getSrcDirectory } from "@latticexyz/common/foundry";
+import { getExistingContracts } from "../utils/getExistingContracts";
+import { getContractData } from "../utils/getContractData";
+import { defaultModuleContracts } from "../utils/defaultModuleContracts";
+import { Hex } from "viem";
+
+type Options = {
+  worldAddress?: string;
+  configPath?: string;
+  profile?: string;
+  srcDir?: string;
+};
+
+const commandModule: CommandModule<Options, Options> = {
+  command: "verify",
+
+  describe: "Verify contracts",
+
+  builder(yargs) {
+    return yargs.options({
+      configPath: { type: "string", desc: "Path to the config file" },
+      profile: { type: "string", desc: "The foundry profile to use" },
+      worldAddress: { type: "string", desc: "Deploy to an existing World at the given address" },
+      srcDir: { type: "string", desc: "Source directory. Defaults to foundry src directory." },
+    });
+  },
+
+  async handler(opts) {
+    const profile = opts.profile ?? process.env.FOUNDRY_PROFILE;
+
+    const configV2 = (await loadConfig(opts.configPath)) as WorldConfig;
+    const config = worldToV1(configV2);
```

Comment:
are there other utils in here that depend on this being the v1 config or could we just use the v2 config in here?

Reply:
we need it to get the user-defined systems, decided to leave as is